packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1318,21 +1337,52 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   setUpLocusInfoSelfListener() {
-    this.locusInfo.on(LOCUSINFO.EVENTS.SELF_MUTED_BY_OTHERS, (payload) => {
-      if (payload) {
+    this.locusInfo.on(LOCUSINFO.EVENTS.LOCAL_UNMUTE_REQUIRED, (payload) => {
+      if (this.audio) {
+        this.audio.handleServerLocalUnmuteRequired(this);
         Trigger.trigger(
           this,
           {
             file: 'meeting/index',
             function: 'setUpLocusInfoSelfListener'
           },
-          EVENT_TRIGGERS.MEETING_SELF_MUTED_BY_OTHERS,
-          {
-            payload
-          }
+          EVENT_TRIGGERS.MEETING_SELF_UNMUTED_BY_OTHERS,
+          payload
```

Comment:
This will break code if someone is expecting an object with a `payload` field in it.

Reply:
yes, you're right, I'll change it back and do same for the other new events I added. Although the "payload" in existing code is not documented at all so not sure if anyone is really using it...

Reply:
done